code/modules/mob/living/abilities/shout.dm

```diff
@@ -0,0 +1,15 @@
+//Simple ability that makes a loud screaming noise, causes screenshake in everyone nearby.
+/mob/proc/shout()
+	set name = "Shout"
+	set category = "Abilities"
+	if (check_audio_cooldown(SOUND_SHOUT))
+		if (play_species_audio(src, SOUND_SHOUT, 100, 1, 2))
+			src.Stun(2)
+			src.shake_animation(40)
+			set_audio_cooldown(SOUND_SHOUT, 8 SECONDS)
+			for (var/mob/M in range(7, src))
```

Comment:
```suggestion
			for (var/mob/M in hearers(7, src))
```

Reply:
nope, hearers is bad, it doesnt actually have anything to do with hearing